src/lsst/cmservice/config.py

```diff
@@ -23,6 +25,7 @@ class Configuration(BaseSettings):
     )
 
     database_password: str | None = Field(
+        default=None,
```

Comment:
We are ping-ponging on this one, too.  It should not be permitted to run postgres/redis without these passwords specified.  What use case are you accommodating here so we can find a way to do that without allowing the running of unauthenticated servers?

Reply:
I think the issue was that the syntax changed 2/ pydatntic 
and this was my effort to get things to run...